ui/perfherder/helpers.js

```diff
@@ -377,3 +389,394 @@ export const getGraphsLink = function getGraphsLink(
+    getData(getApiUrl(`/performance/alertsummary/${id}/`)).then(({ data }) =>
+      AlertSummary(data, optionCollectionMap),
+    ),
+  );
+
+export const getAlertSummaryTitle = id =>
+  getAlertSummary(id).then(alertSummary => getTitle(alertSummary));
+
+export const getAlertSummaries = options => {
+  let { href } = options;
+  if (!options || !options.href) {
+    href = getApiUrl('/performance/alertsummary/');
+
+    // add filter parameters for status and framework
+    const params = [];
+    if (
+      options &&
+      options.statusFilter !== undefined &&
+      options.statusFilter !== -1
+    ) {
+      params[params.length] = `status=${options.statusFilter}`;
+    }
+    if (options && options.frameworkFilter !== undefined) {
+      params[params.length] = `framework=${options.frameworkFilter}`;
+    }
+    if (options && options.signatureId !== undefined) {
+      params[params.length] = `alerts__series_signature=${options.signatureId}`;
+    }
+    if (options && options.seriesSignature !== undefined) {
+      params[params.length] = `alerts__series_signature__signature_hash=${
+        options.seriesSignature
+      }`;
+    }
+    if (options && options.repository !== undefined) {
+      params[params.length] = `repository=${options.repository}`;
+    }
+    if (options && options.page !== undefined) {
+      params[params.length] = `page=${options.page}`;
+    }
+
+    if (params.length) {
+      href += `?${params.join('&')}`;
+    }
+  }
+
+  return OptionCollectionModel.getMap().then(optionCollectionMap =>
+    getData(href).then(({ data }) => ({
+      results: data.results.map(alertSummaryData =>
+        AlertSummary(alertSummaryData, optionCollectionMap),
+      ),
+      next: data.next,
+      count: data.count,
+    })),
+  );
+};
+
+export const createAlert = data =>
+  create(getApiUrl('/performance/alertsummary/'), {
+    repository_id: data.project.id,
+    framework_id: data.series.frameworkId,
+    push_id: data.resultSetId,
+    prev_push_id: data.prevResultSetId,
+  }).then(({ data }) => {
+    const newAlertSummaryId = data.alert_summary_id;
+    return create(getApiUrl('/performance/alert/'), {
+      summary_id: newAlertSummaryId,
+      signature_id: data.series.id,
+    }).then(() => newAlertSummaryId);
+  });
+
+export const findPushIdNeighbours = (dataPoint, resultSetData, direction) => {
+  const pushId = dataPoint.resultSetId;
+  const pushIdIndex =
+    direction === 'left'
+      ? resultSetData.indexOf(pushId)
+      : resultSetData.lastIndexOf(pushId);
+  const relativePos = direction === 'left' ? -1 : 1;
+  return {
+    push_id: resultSetData[pushIdIndex + relativePos],
+    prev_push_id: resultSetData[pushIdIndex + (relativePos - 1)],
+  };
+};
+
+export const nudgeAlert = (dataPoint, towardsDataPoint) => {
+  const alertId = dataPoint.alert.id;
+  return update(getApiUrl(`/performance/alert/${alertId}/`), towardsDataPoint);
+};
+
+export { AlertSummary };
```

Comment:
Could you remove this export statement and instead export `AlertSummary` at the same time as defining it?